doc/cookbook/compression_threads.cpp

```diff
@@ -2,17 +2,17 @@
 //![example]
 #include <seqan3/io/all.hpp>
 
-
-// The `bgzf_thread_count` is a variable that can only be changed during run time of the program.
-// This does not work, the value must be overwritten within a function.
-// seqan3::contrib::bgzf_thread_count = 1u; // Doesn't work
+// The `bgzf_thread_count` is a variable that can only be changed during the runtime of a program.
+// The following does not work, the value must be overwritten within a function:
```

Comment:
```suggestion
// The following does not work, because the value must be overwritten within a function:
```

Reply:
Hmm, not sure. It depends on how you parse it. Does it not work because it needs to be overwritten in a function? Or it does not work; hence we need to overwrite it in a function?

I tend to just leave it as is.